util/missingcrypto.txt

```diff
@@ -636,6 +636,7 @@ ERR_load_KDF_strings(3)
 ERR_load_OBJ_strings(3)
 ERR_load_OCSP_strings(3)
 ERR_load_OSSL_SERIALIZER_strings(3)
+ERR_load_OSSL_DESERIALIZER_strings(3)
```

Comment:
Why does this need to be public?

Reply:
It's autodefined by mkerr.pl.  This question gets asked *every time* :wink:

Reply:
And we still seem to be making them public unnecessarily. If the script is the problem, we should fix the script....but yeah ok...out  of scope of this PR...sigh.